.github/workflows/ci.yml

```diff
@@ -10,5 +10,27 @@ jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-    - name: Dummy CI
-      run: echo "OK"
+      - uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Build and cache base image
+        uses: docker/bake-action@v6
+        with:
+          push: false
+          load: true
+          files: compose.yml
+          targets: base
+          set: |
+            base.cache-from=type=gha,scope=base-cached-tests
+            base.cache-to=type=gha,scope=base-cached-tests,mode=max
+
```

Comment:
Can we add a step to invoke the equivalent of `make lint` here, before we bother starting Postgres?